client/rpc.go

```diff
@@ -260,3 +277,14 @@ func (s *Subscription) BalanceProof(ctx context.Context) (tee.BalanceProof, erro
 		return proof, nil
 	}
 }
+
+// TxReceipt blocks until it can return the next pending incoming transaction
+// from the operator or an error if the context ran out.
```

Comment:
I find "or an error if the context ran out." redundant. We'd need to write it everywhere, every time, otherwise. It's usual behavior.